magento/pwa-studio
# [doc] intercepting talons tutorial

## Description

Creates an overview topic and tutorial for intercepting talons

## Related Issue

Closes PWA-799

## Acceptance 

Anyone in the core team

### Verification Stakeholders

Anyone in the core team

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
1. Start the html preview server: `yarn develop`
1. Navigate to the tutorials section
1. Verify **Intercept a Target** is listed on the left nav
1. Verify **Modify talon results** is listed underneath
1. Verify links leads to expected pages

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
